packages/atlas/src/AnalyticsManager.tsx

```diff
@@ -29,7 +29,6 @@ export const AnalyticsManager: FC = () => {
 
     ls.init(atlasConfig.analytics.livesession.id, {
       keystrokes: true,
-      rootHostname: atlasConfig.analytics.livesession.rootHostname,
```

Comment:
If we don't need it we should remove this from the atlas config and from the config schema.